capitulo 1/lang_en.json

```diff
@@ -3705,46 +3705,46 @@
+  "obj_npc_facing_slash_Other_10_gml_611_0": "* Sim!^1! E estou mudando algumas regras grandes!/",
   "obj_npc_facing_slash_Other_10_gml_612_0": "\\E1* Primeira^1, Iluminados agora são permitidos!/",
   "obj_npc_facing_slash_Other_10_gml_613_0": "\\E2* Segunda^1, prisão agora é apenas para pais!/",
-  "obj_npc_facing_slash_Other_10_gml_615_0": "* Soa como algumas mudanças muito boas./",
+  "obj_npc_facing_slash_Other_10_gml_615_0": "* Parece boas mudanças./",
   "obj_npc_facing_slash_Other_10_gml_616_0": "\\E1* Ei, seu pai estava^1, ham^1, protegendo a fonte^1, certo?/",
-  "obj_npc_facing_slash_Other_10_gml_617_0": "\\E0* ... vai ficar tudo bem se mexermos...?/",
+  "obj_npc_facing_slash_Other_10_gml_617_0": "\\E0* ... será que vai ficar tudo bem se mexermos nela...?/",
   "obj_npc_facing_slash_Other_10_gml_619_0": "* Eu não faço ideia!/",
-  "obj_npc_facing_slash_Other_10_gml_620_0": "\\E2* Mas é vocês caras^1, então eu confio em vocês...!/",
-  "obj_npc_facing_slash_Other_10_gml_621_0": "\\E3* Se algo ruim acontecer^1, vamos consertar!/",
-  "obj_npc_facing_slash_Other_10_gml_623_0": "* É^1, acho que vamos ver disso eu acho./",
+  "obj_npc_facing_slash_Other_10_gml_620_0": "\\E2* Mas já que serão vocês^1, então eu posso confiar...!/",
+  "obj_npc_facing_slash_Other_10_gml_621_0": "\\E3* Se algo ruim acontecer^1, a gente conserta!/",
+  "obj_npc_facing_slash_Other_10_gml_623_0": "* É^1, só vendo pra saber eu acho./",
   "obj_npc_facing_slash_Other_10_gml_624_0": "\\E2* Hm..^1. bem^1, te vejo depois!/",
-  "obj_npc_facing_slash_Other_10_gml_626_0": "* Volte logo!!!/",
+  "obj_npc_facing_slash_Other_10_gml_626_0": "* Voltem logo!!!/",
   "obj_npc_facing_slash_Other_10_gml_627_0": "\\E2* Sempre tem lugar na cidade^1, pra uma dupla de palhaços!/%",
   "obj_npc_facing_slash_Other_10_gml_636_0": "* Você deveria dizer ao seu Príncipe que eu disse tchau também./",
   "obj_npc_facing_slash_Other_10_gml_638_0": "\\E3* ... ele tá saindo?/",
-  "obj_npc_facing_slash_Other_10_gml_640_0": "* Não..^1. ele só é um docinho./%",
+  "obj_npc_facing_slash_Other_10_gml_640_0": "* Não..^1. ele é um doce, só isso./%",
   "obj_npc_room_animated_slash_Other_10_gml_83_0": "* (Sua expressão anteriormente violenta se tornou bondosa...)/",
-  "obj_npc_room_animated_slash_Other_10_gml_84_0": "* (Agora vai viver uma vida tranquila.)/%",
-  "obj_npc_room_animated_slash_Other_10_gml_89_0": "* (Porque você salvou ele^1, cresceu para te respeitar...)/",
-  "obj_npc_room_animated_slash_Other_10_gml_90_0": "* (Bem^1, na verdade^1, talvez encolheu para te respeitar.)/%",
+  "obj_npc_room_animated_slash_Other_10_gml_84_0": "* (P.Circular viverá uma vida tranquila.)/%",
+  "obj_npc_room_animated_slash_Other_10_gml_89_0": "* (Já que você o salvou^1, seu respeito por você cresceu...)/",
+  "obj_npc_room_animated_slash_Other_10_gml_90_0": "* (Bem^1, na verdade^1, ele encolheu mas ainda respeita você mesmo assim.)/%",
```

Comment:
Aqui eu não expliquei muito bem o porquê de não querer essas mudanças.

Por qual motivo mudar _salvou ele_ pra _o salvou_? O ReiDondo continua tendo gênero assim.

Sobre as outras duas partes (_seu respeito..._ e _mais ainda..._), nisso você além de mudar o sentido da segunda frase destruiu o jogo de palavras. Observe como é em Inglês:

(Because you rescued it, _it grew to respect you..._)
(Well, actually, _**maybe** it shrunk to respect you._)

Tem um jogo de palavras, semelhante a uma rima, e você negligenciou isso. Além disso, parece que você não notou o _maybe_, que representa dúvida, e o _na verdade/actually_ serve como uma correção.
Por esses motivos eu não aceito essas mudanças, tirando o _Já que_, pois isso é uma melhoria.